csharp-selenium-webdriver-sample/WebDriverFactory.cs

```diff
@@ -21,13 +21,29 @@ public static IWebDriver GetWebDriver(BrowserType browser)
             switch (browser)
             {
                 case BrowserType.Chrome:
-                    // Check if Chrome web driver environment variable already defined so that Chrome version and the webdriver are the same, otherwise use Selenium.WebDriver.ChromeDriver to install it                    
+                    // So now we need to specify the ChromeDriver directory path
```

Comment:
Consider updating this to refer to the pipelines docs about the environment variable and to focus more on the reasoning for *why* we're using them. Perhaps something like this?

```cs
// The ChromeWebDriver environment variable comes pre-set in the Azure Pipelines VM Image we
// specify in azure-pipelines.yml. ChromeDriver requires that you use *matching* versions of Chrome and
// the Chrome WebDriver; in the build agent VMs, using this environment variable will make sure that we use
// the pre-installed version of ChromeDriver that matches the pre-installed version of Chrome.
//
// See https://docs.microsoft.com/en-us/azure/devops/pipelines/test/continuous-test-selenium
//
// Environment.CurrentDirectory is where the Selenium.Webdriver.ChromeDriver NuGet package will place the
// version of the Chrome WebDriver that it comes with. We fall back to this so that if a developer working on
// the project hasn't separately installed ChromeDriver, the test will still be able to run on their machine.
```